src/MiNET/MiNET/Player.cs

```diff
@@ -2236,6 +2294,23 @@ public void HandleMcpeSubChunkRequestPacket(McpeSubChunkRequestPacket message)
 			SendPacket(response);*/
 		}
 
+		public virtual void HandleMcpeRequestAbility(McpeRequestAbility message)
+		{
+			if (message.ability == 18) // flying??
```

Comment:
I have doubts this is correct

Reply:
Is it important to fix this as part of this PR?